packages/parser-webpack-config/README.md

```diff
@@ -0,0 +1,71 @@
+# Parser webpack-config (`@sonarwhal/parser-webpack-config`)
+
+The `webpack-config` parser allows the user to analyze the
+Webpack configuration in their projects.
```

Comment:
> webpack 

(It seems [they](https://github.com/webpack/webpack) use `webpack` instead of `Webpack`)

Can you make this change everywhere? Thanks!